Popular_Models_Guide/StableDiffusion/README.md

```diff
@@ -0,0 +1,300 @@
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Deploying Stable Diffusion Models with Triton and TensorRT
+
+This example demonstrates how to deploy Stable Diffusion models in
+Triton by leveraging the [TensorRT demo](https://github.com/NVIDIA/TensorRT/tree/release/9.2/demo/Diffusion)
+pipeline and utilities.
+
+Using the TensorRT demo as a base this example contains a reusable
+[python based backend](https://github.com/triton-inference-server/backend/blob/main/docs/python_based_backends.md), [`/backend/diffusion/model.py`](backend/diffusion/model.py),
+suitable for deploying multiple versions and configurations of
+Diffusion models.
+
+For more information on Stable Diffusion please visit
+[stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5),
+[stable-diffusion-xl](https://huggingface.co/docs/diffusers/en/using-diffusers/sdxl). For
+more information on the TensorRT implementation please see the [TensorRT demo](https://github.com/NVIDIA/TensorRT/tree/release/9.2/demo/Diffusion).
+
+> [!Note]
+> This example is given as sample code and should be reviewed before use in production settings.
+
+| [Requirements](#requirements) | [Building Server Image](#building-the-triton-inference-server-image) | [Stable Diffusion v1.5](#building-and-running-stable-diffusion-v-15) | [Stable Diffusion XL](#building-and-running-stable-diffusion-xl) | [Sending an Inference Request](#sending-an-inference-request) | [Model Configuration](docs/model_configuration.md) | [Sample Client](#sample-client) | [Known Issues and Limitations](#known-issues-and-limitations) |
+
+## Requirements
+
+The following instructions require a Linux system with Docker
+installed. For CUDA support, make sure your CUDA driver meets the
+requirements in the "NVIDIA Driver" section of [Deep Learning Framework
+support matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html).
+
+## Building the Triton Inference Server Image
+
+The example is designed based on the
+`nvcr.io/nvidia/tritonserver:24.01-py3` docker image and [TensorRT OSS v9.2.0](https://github.com/NVIDIA/TensorRT/releases/tag/v9.2.0).
+
+A set of convenience scripts are provided to create a docker image
+based on the `nvcr.io/nvidia/tritonserver:24.01-py3` image with the
+dependencies for the TensorRT Stable Diffusion demo installed.
+
+### Triton Inference Server + TensorRT OSS
+
+#### Clone Repository
+```bash
+git clone https://github.com/triton-inference-server/tutorials.git -b r24.02 --single-branch
+cd tutorials/Popular_Models_Guide/StableDiffusion
```

Comment:
I think saying "This is based on 24.01" and "git clone 24.02" may confuse people. Can we just use `main` branch for this clone?
```suggestion
git clone https://github.com/triton-inference-server/tutorials.git -b r24.02 --single-branch
cd tutorials/Popular_Models_Guide/StableDiffusion
```